docs/pages/database-access/reference/cli.mdx

```diff
@@ -40,17 +40,17 @@ $ teleport db start \
 
 ## tctl auth sign
 
-When invoked with a `--format=db` flag, produces a CA certificate, a client
-certificate and a private key file used for configuring Database Access with
-self-hosted database instances.
+When invoked with a `--format=db` (or `--format=mongodb` for MongoDB) flag,
+produces a CA certificate, a client certificate and a private key file used for
+configuring Database Access with self-hosted database instances.
 
 ```shell
-tctl auth sign --format=tls --user=access-plugin-mattermost --out=auth --ttl=2190h
+$ tctl auth sign --format=db --host=db.example.com --out=db --ttl=8760h
```

Comment:
@inertial-frame FYI this command was incorrectly changed in https://github.com/gravitational/teleport/pull/7158, I reverted it back so it makes sense for database access.

Reply:
After some initial discussion, we agreed on using 3 months for our examples requiring long-lived certs. Confirming that we want all DB examples to be `8760h`? 

May be a few places that we'd want to revert that.

Reply:
Hmm, I'm okay changing it though I'm wondering if 3 months may be a bit too aggressive for databases. Effectively means they would have to rotate secrets for all their dbs every 3 months which may be annoying (esp. since updating database certs requires restarting the database).

As an option we could change to 3 months as a recommendation (they can always choose their own TTL after all) and add a note with what I wrote above.

Reply:
@r0mant - Yeah that makes sense. Anecdotally, rotating **6 months** or **annually** seems fairly common.

I don't believe GDPR or other compliance standards typically define a rotation lifecycle or timeframe. A quick bit of research shows **6 months** or **1 year** as fairly common.

We're adding an article that will be dedicated to short-lived TTL (excluding DB's and other mainstays) which will include basic commands for inspecting, revoking, renewing, etc. (we can then point to that article/section throughout the docs).

I'm wondering if for these long-lived certs if it's better to use a timeframe that aligns with the above (probably anywhere between **3 months to a year**, with a slight preference for shorter durations) but note that these are examples. Maybe this could be moved into an include as an Admonition?

Reply:
@inertial-frame Okay, I changed it back to 2160h and added a note about picking TTL.